diff -Naur ./cube-tcm_old/module/vtcmd_dev/vtcmd_dev.c ./cube-tcm/module/vtcmd_dev/vtcmd_dev.c
--- ./cube-tcm_old/module/vtcmd_dev/vtcmd_dev.c	2019-04-15 18:26:01.726774000 +0800
+++ ./cube-tcm/module/vtcmd_dev/vtcmd_dev.c	2019-04-15 18:29:26.546262745 +0800
@@ -170,8 +170,8 @@
   int res;
   struct sockaddr_in * tcm_addr;
 //   res = sock_create(PF_UNIX, SOCK_STREAM, 0, &tcmd_sock);
-     res = sock_create_kern(AF_INET, SOCK_STREAM, 0, &tcmd_sock);
- //  res = sock_create_kern(&init_net,AF_INET, SOCK_STREAM, 0, &tcmd_sock);
+//   res = sock_create_kern(AF_INET, SOCK_STREAM, 0, &tcmd_sock);
+     res = sock_create_kern(&init_net,AF_INET, SOCK_STREAM, 0, &tcmd_sock);
   if (res != 0) {
     //error("sock_create_kern() failed: %d\n", res);
     tcmd_sock = NULL;
